src/sdl2/audio.rs

```diff
@@ -263,6 +268,7 @@ impl ExactSizeIterator for DriverIterator { }
 
 /// Gets an iterator of all audio drivers compiled into the SDL2 library.
 #[inline]
+#[doc(alias = "SDL_GetNumAudioDrivers")]
```

Comment:
Like `SDL_GetVideoDriver` `SDL_GetAudioDriver` should be probably be here instead.